tests/tests/test-reduced-permissions.spec.js

```diff
@@ -251,11 +251,13 @@ context('Test reduced permissions', () => {
   it('Cleanup', () => {
     cy.get('[data-testid="app-switcher"]').click();
 
-    // 2 results: original cluster and "Clusters Overview" node, take first
+    // 2 results: "Clusters Overview" node and original cluster, take second
     cy.get('#appSwitcherPopover:visible')
+      .find('li')
+      .eq(1)
       .find('[role="button"]')
-      .first()
       .click();
+    cy.wait(1000);
 
     // wait until original cluster loads
     cy.getIframeBody()
```

Comment:
Instead of wait(1000), we can reverse a test to check if Configuration is back in the menu,
```suggestion
       cy.getLeftNav()
      .contains('Configuration')
      .should('exist');
```